### PR TITLE
Cleans up end of round robo report

### PR DESCRIPTION
**Pull request by ShizCalev**
Making it a little easier to read by inserting some linebreaks and bolding player names/ckeys instead of entire lines. Also cleaned up some rogue spacing and commas.

Old:
old

new:
new 2
new 3
new 1

also if anyone has any suggestions on how to indent the laws, that'd be cool too tbh.

### CODE REVIEW

**Comment by ShizCalev** [comment]
If anyone has any suggestions on how to indent the laws, that'd be awesome. Not sure how it'd work with the list and all that.

File: code/__HELPERS/roundend.dm [hunk 1]
```diff
@@ -285,6 +285,7 @@
 			var/list/ded = SSblackbox.first_death
 			if(ded.len)
 				parts += "[GLOB.TAB]First Death: <b>[ded["name"]], [ded["role"]], at [ded["area"]]. Damage taken: [ded["damage"]].[ded["last_words"] ? " Their last words were: \"[ded["last_words"]]\"" : ""]</b>"
+			//fukken syntax parsing memes"
```
Comment (vuonojenmustaturska) [inline code/__HELPERS/roundend.dm]: Please don't add comments that someone else will have to remove later

Comment (vuonojenmustaturska) [inline code/__HELPERS/roundend.dm]: please don't leave comments that someone else will have to remove later

**Review by vuonojenmustaturska — CHANGES_REQUESTED** [review]

File: code/__HELPERS/roundend.dm (hunk 1 above)
Comment (ShizCalev) [reply code/__HELPERS/roundend.dm]: It's not useless. The \" in the line directly above it isn't being parsed correctly, resulting in the rest of the file being fubar'd on visual code.
image

image

Comment (vuonojenmustaturska) [reply code/__HELPERS/roundend.dm]: welp

**Review by vuonojenmustaturska — DISMISSED** [review]

**Review by vuonojenmustaturska — APPROVED** [review]

**Pull request merged**